core/kazoo_documents/src/kzd_freeswitch.erl

```diff
@@ -52,11 +52,14 @@
 
 -include("kz_documents.hrl").
 
+-type data() :: kz_proplist().
```

Comment:
Whatever the value of doc/0 we should go with it. That's why we have an alias type.

Agreed on autogen kzd_freeswitch